web/src/lib/provider/session.js

```diff
@@ -35,6 +37,14 @@ const AuthProvider = ({ children }) => {
   const searchParams = useSearchParams()
   const paramsSize = [...searchParams.keys()].length
 
+  const isIdle = useIdle(18e5) // 30 minutes
```

Comment:
What's scope of action will broke the idle? And the `initialState` is settled, Can we use the configurable value.

Reply:
Please check [useIdle(react-use)](https://github.com/streamich/react-use/blob/master/src/useIdle.ts).
The default events list is 'mousemove', 'mousedown', 'resize', 'keydown', 'touchstart', 'wheel'.